omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -13,18 +8,81 @@ Nginx Gunicorn configuration (Unix/Linux)
       see :ref:`python-requirements` on the
       :doc:`/sysadmins/version-requirements` page.
 
-Install `Django 1.8`_ and `Gunicorn <http://docs.gunicorn.org/>`_ using
-package requirements file:
+.. _gunicorn_default_configuration:
+
+Gunicorn default configuration (Unix/Linux)
+-------------------------------------------
+
+Install `Django 1.8`_,
+`Gunicorn >= 19.3 <http://gunicorn.org>`_
+and `futures <https://pypi.python.org/pypi/futures>`_ using package requirements file:
```

Comment:
corresponding PR given in the description https://github.com/openmicroscopy/openmicroscopy/pull/4447/files#diff-726e917f2a5b8715f138941e647d150fR8